client/app/resources/mock-data.factory.js

```diff
@@ -0,0 +1,431 @@
+(function() {
+  'use strict';
+
+  angular.module('app.resources')
+    .factory('MockData', MockDataFactory);
+
+  /** @ngInject */
+  function MockDataFactory() {
+    var service = {
+      retrieveMockData: retrieveMockData
+    };
+
+    return service;
+
+    function retrieveMockData(collection) {
+      if (collection === 'service_catalogs') {
```

Comment:
@dtaylor113 instead of mocking this here you could have used existing REST calls to fetch service_templates and same for list of service_dialogs.